assets/onboarding/features/confirmation-modal.jsx

```diff
@@ -17,11 +17,19 @@ const WC_EXTRA_DESCRIPTION = __(
  * Features confirmation modal.
  *
  * @param {Object}    props
- * @param {Feature[]} props.features  Features list.
- * @param {Function}  props.onInstall Callback to install the features.
- * @param {Function}  props.onSkip    Callback to skip the installation.
+ * @param {Feature[]} props.features      Features list.
+ * @param {boolean}   props.isSubmitting  Is submitting state.
+ * @param {Element}   [props.errorNotice] Is submitting state.
```

Comment:
This has some extra brackets.

Reply:
It's because it's an optional parameter.

Reply:
TIL 🤓